potato_client/src/App.js

```diff
@@ -176,8 +185,7 @@ class AccommodationItem extends Component {
             </span>
```

Comment:
I was looking into how, in JavaScript, to accomplish the goal here of putting the slashes only between the items, and I found a [StackOverflow Question](https://stackoverflow.com/questions/34034038/how-to-render-react-components-by-using-map-and-join) that is asking this exact thing (Google search was "react map join", because .join() is the list method you'd normally use to combine a string with stuff in between but not on the end).

It looks like we could do something like:

```javascript
render() {
  <div>
    {
      this.props.links
        .map(link => <AccommodationLink url={link.url} title={link.title}/>)
        .reduce((prev, curr) => [prev, ' / ', curr])
    }
  </div>
}
```

want to give that a shot before we merge this?